pytorch_lightning/accelerators/accelerator.py

```diff
@@ -66,9 +66,9 @@ def setup(self, trainer, model: LightningModule) -> None:
             trainer: the trainer instance to connect to
             model: the model to train
         """
-        self.connect_training_type_plugin(self.training_type_plugin, model)
+        self.connect_training_type_plugin(model)
```

Comment:
The changes in this file are not relevant for the PR, are they?

Reply:
already reverted locally. was checking something.